tests/python/unittest/test_gluon_data.py

```diff
@@ -78,7 +78,6 @@ def _dataset_transform_fn(x, y):
     return x, y
 
 @with_seed()
-@unittest.skip("Flaky test: https://github.com/apache/incubator-mxnet/issues/13484")
```

Comment:
Should this be here?  Looks like you have a rogue commit

Reply:
Yes, it fixes the "flaky" test which isn't flaky.